packages/mds-policy-service/service/helpers.ts

```diff
@@ -10,11 +10,17 @@ import type {
 } from '../@types/intents'
 import { BASE_POLICY_DEFAULTS, INTENT_RULE_CONSTANTS } from '../@types/intents'
 
+export const TWENTY_MINUTES = 1000 * 60 * 20
```

Comment:
I'd just use the `minutes` helper from `mds-utils` for this, I dunno if it's necessary to make it a constant either